### PR TITLE
Add the hint LIMIT_TO_COP()

### PR DESCRIPTION
**Pull request by qw4990**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

Add the hint LIMIT_TO_COP()

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by winoros — APPROVED** [review]

File: optimizer-hints.md [hunk 1]
```diff
@@ -183,6 +183,17 @@ SELECT /*+ IGNORE_INDEX(t1, idx1, idx2) */ * FROM t t1;
 SELECT /*+ AGG_TO_COP() */ sum(t1.a) FROM t t1;
 ```
 
+### LIMIT_TO_COP()
+
+`LIMIT_TO_COP()` 提示优化器将制定查询快中的 Limit 和 TopN 算子下推到 coprocessor。如果优化器没有下推 Limit 或者 TopN 算子，建议尝试。例如：
```
Comment (ichn-hu) [inline optimizer-hints.md:188]:
```suggestion
`LIMIT_TO_COP()` 提示优化器将制定查询块中的 Limit 和 TopN 算子下推到 coprocessor。优化器没有下推 Limit 或者 TopN 算子时建议尝试使用该提示。例如：
```

**Review by ichn-hu — APPROVED** [review]
Rest LGTM

File: optimizer-hints.md [hunk 2]
```diff
@@ -183,6 +183,17 @@ SELECT /*+ IGNORE_INDEX(t1, idx1, idx2) */ * FROM t t1;
 SELECT /*+ AGG_TO_COP() */ sum(t1.a) FROM t t1;
 ```
 
+### LIMIT_TO_COP()
+
+`LIMIT_TO_COP()` 提示优化器将制定查询块中的 Limit 和 TopN 算子下推到 coprocessor。优化器没有下推 Limit 或者 TopN 算子时建议尝试使用该提示。例如：
```
Comment (TomShawn) [inline optimizer-hints.md:188]:
```suggestion
`LIMIT_TO_COP()` 提示优化器将指定查询块中的 `Limit` 和 `TopN` 算子下推到 coprocessor。优化器没有下推 Limit 或者 TopN 算子时建议尝试使用该提示。例如：
```

File: optimizer-hints.md [hunk 3]
```diff
@@ -183,6 +183,17 @@ SELECT /*+ IGNORE_INDEX(t1, idx1, idx2) */ * FROM t t1;
 SELECT /*+ AGG_TO_COP() */ sum(t1.a) FROM t t1;
 ```
 
+### LIMIT_TO_COP()
+
+`LIMIT_TO_COP()` 提示优化器将指定查询块中的 `Limit` 和 `TopN` 算子下推到 coprocessor。优化器没有下推 Limit 或者 TopN 算子时建议尝试使用该提示。例如：
```
Comment (TomShawn) [inline optimizer-hints.md:188]:
```suggestion
`LIMIT_TO_COP()` 提示优化器将指定查询块中的 `Limit` 和 `TopN` 算子下推到 coprocessor。优化器没有下推 `Limit` 或者 `TopN` 算子时建议尝试使用该提示。例如：
```

**Pull request merged**